images/process/workflow-runner/module.bash

```diff
@@ -29,14 +29,22 @@ while read -r line; do
   namespace=$(echo "${DATA_JSON}" | jq -r .namespace)
   environment=$(echo "${DATA_JSON}" | jq -r .environment)
   team=$(echo "${DATA_JSON}" | jq -r .team)
-  IMAGE_NAME=$(echo "${DATA_JSON}" | jq -r .image)
+  IMAGE=$(echo "${DATA_JSON}" | jq -r .image)
   IMAGE_ID=$(echo "${DATA_JSON}" | jq -r .image_id)
   export IMAGE_ID #used in parse_and_set_image_variables
+  export IMAGE #used in parse_and_set_image_variables
   parse_and_set_image_variables
+  
+  
   appname=$(echo "${DATA_JSON}" | jq -r .app_kubernetes_io_name)
+  
   if [ "${appname}" == "" ] || [ "${appname}" == "null" ]; then
+    #IMAGE_NAME is exportet from parse_and_set_image_variables()
     appname="${IMAGE_NAME}"
-    echo "app_kubernetes_io_name is empty, setting to: ${IMAGE_NAME}"
+    appname="${appname%@*}"
+    appname="${appname%:*}"
```

Comment:
Wieso ohne registry? Bei mehreren Registries könnte es verwirrend sein nicht zu wissen, wo es her kommt.

tpagel@desktop:~$ IMAGE="internal.organization:5000/user/image:x"
tpagel@desktop:~$ source git/sda/cluster-image-scanner/images/base/scan-common.bash 
IMAGE: internal.organization:5000/user/image:x, IMAGE_ID: 
tpagel@desktop:~$ echo $IMAGE
internal.organization:5000/user/image:x
tpagel@desktop:~$     appname="${IMAGE_NAME}"
    appname="${appname%@*}"
    appname="${appname%:*}"
    appname="${appname#*/}"
tpagel@desktop:~$ echo $appname
**user/image**




tpagel@desktop:~$ IMAGE="internal.organization/user/image:x"
tpagel@desktop:~$ parse_and_set_image_variables
tpagel@desktop:~$ echo $IMAGE_NAME
internal.organization/user/image
tpagel@desktop:~$     appname="${IMAGE_NAME}"
    appname="${appname%@*}"
    appname="${appname%:*}"
tpagel@desktop:~$  appname="${appname#*/}"
    echo "app_kubernetes_io_name is empty, setting to: ${appname}"
app_kubernetes_io_name is empty, setting to: **user/image**